data-raw/knights_landing/knights_landing_modeling.R

```diff
@@ -0,0 +1,148 @@
+library(tidyverse)
+library(lubridate)
+library(dataRetrieval)
+library(CDECRetrieve)
+library(forcats)
+library(rnoaa)
+library(caret)
+
+# hourly water temperature data on VON (fahrenheit)
+#07/29/1998 to present
+
+CDECRetrieve::cdec_stations("VON")
```

Comment:
What are the time serries on these stations. We should confirm that time they do not have coverage in the 1979 - 2000 period - if they do we can just use gage data directly and interpolate missing months.

Reply:
Lets talk through this tomorrow

Reply:
There is a USGS gage at Knights Landing that has water temperature data but only for a year from 1995 to 1996 https://waterdata.usgs.gov/ca/nwis/inventory/?site_no=11391100